### PR TITLE
W-15428118-Accelerator for Service Cloud

### PR DESCRIPTION
**Pull request by nisha-nizamuddin**

### CODE REVIEW

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+= Object Mappings
+
+== Customer profile sync
+
+=== Source-type mappings
+
+The following table summarizes how representations of a customer or individual from each system will be mapped to CIM types:
+
+[%header%autowidth.spread]
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc:9]: check table formatting

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
+= Object Mappings
+
+== Customer profile sync
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
== Customer Profile Sync
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+=== Source-type mappings
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
=== Source-Type Mappings
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+The following table summarizes how representations of a customer or individual from each system will be mapped to CIM types:
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
The following table shows how customers or individuals from each system are mapped to CIM types:
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+[%header%autowidth.spread]
+|===
+| Source System | Source Type | CIM Types | Mapping Notes
+
+| Salesforce
+Service Cloud | Person Accounts | Individual
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
Service Cloud | Person Accounts | * Individual
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+CustomerContactAddress
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* CustomerContactAddress
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Email
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Email
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Phone | Person Account is represented as record type, mapping to Account and Contact
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Phone | A Person Account is represented as a record type that maps to Account and Contact.
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+| SAP | Customer | IndividualCustomer
+
+Contact Address
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Address
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Email
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Email
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Phone |
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Phone |
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 12]
```diff
@@ -0,0 +1,61 @@
...
+| Salesforce
+Service Cloud | Person Accounts | Individual
+
+CustomerContactAddress
+
+Contact Email
+
+Contact Phone | Person Account is represented as record type, mapping to Account and Contact
+
+| SAP | Customer | IndividualCustomer
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
| SAP | Customer | * IndividualCustomer
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 13]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Address
+
+Contact Email
+
+Contact Phone |
+|===
+
+=== Target-type mappings
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
=== Target-Type Mappings
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+
+The following table summarizes how representations of a customer or individual from each system will be mapped from CIM types:
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
The following table shows how customers or individuals from each system are mapped to CIM types:
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 15]
```diff
@@ -0,0 +1,61 @@
...
+
+[%header%autowidth.spread]
+|===
+| Target System | CIM Type | Target Types | Mapping Notes
+
+| Salesforce
+Service Cloud | CustomerIndividual
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
Service Cloud | * CustomerIndividual
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Address
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Address
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 17]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Email
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Email
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Phone | PersonAccounts | Relevant fields only
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Phone | PersonAccounts | Relevant fields only
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+
+| SAP | CustomerIndividual
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
| SAP | * CustomerIndividual
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Address
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Address
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Email
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Email
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+
+Contact Phone | Customer | Relevant fields only
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* Contact Phone | Customer | Relevant fields only
```

File: service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+|===
+
+
+== See Also 
+
+* xref:prerequisites.adoc[Prerequisites]
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/cim-mappings.adoc]:
```suggestion
* xref:prerequisites.adoc[Before You Begin]
```

File: service-cloud/1.0/modules/ROOT/pages/index.adoc [hunk 24]
```diff
@@ -1,23 +1,14 @@
-= MuleSoft Accelerator for Retail
-:retail-version: {page-component-version}
+= MuleSoft Accelerator for Service Cloud
+:service-cloud-version: {page-component-version}
 
-MuleSoft Accelerator for Retail empowers retail IT teams to unlock data from systems including ERP, CRM, MDM, commerce, and marketing to activate digital projects faster. Retail IT teams can take advantage of pre-built APIs, connectors, implementation templates, and reference architecture to create customer 360 profiles.
+MuleSoft Accelerator for Service Cloud empowers service representatives with a holistic customer 360 view, facilitating personalized interactions, optimized service workflows, and heightened operational efficiency, all aimed at enhancing customer satisfaction.
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
MuleSoft Accelerator for Service Cloud empowers service representatives with a holistic customer 360 view, facilitating personalized interactions, optimized service workflows, and heightened operational efficiency, to enhance customer satisfaction.
```

File: service-cloud/1.0/modules/ROOT/pages/index.adoc [hunk 25]
```diff
@@ -1,23 +1,14 @@
...
 
 == Use Cases
 
-The following use cases are supported by the MuleSoft Accelerator for Retail:
+The following use cases are supported by the MuleSoft Accelerator for Service Cloud:
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
The following use case is supported by the MuleSoft Accelerator for Service Cloud:
```

File: service-cloud/1.0/modules/ROOT/pages/index.adoc [hunk 26]
```diff
@@ -1,23 +1,14 @@
-= MuleSoft Accelerator for Retail
-:retail-version: {page-component-version}
+= MuleSoft Accelerator for Service Cloud
+:service-cloud-version: {page-component-version}
 
-MuleSoft Accelerator for Retail empowers retail IT teams to unlock data from systems including ERP, CRM, MDM, commerce, and marketing to activate digital projects faster. Retail IT teams can take advantage of pre-built APIs, connectors, implementation templates, and reference architecture to create customer 360 profiles.
+MuleSoft Accelerator for Service Cloud empowers service representatives with a holistic customer 360 view, facilitating personalized interactions, optimized service workflows, and heightened operational efficiency, all aimed at enhancing customer satisfaction.
 
 == Use Cases
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
== Use Case
```

File: service-cloud/1.0/modules/ROOT/pages/index.adoc [hunk 27]
```diff
@@ -1,23 +1,14 @@
...
 
-The following use cases are supported by the MuleSoft Accelerator for Retail:
+The following use cases are supported by the MuleSoft Accelerator for Service Cloud:
 
-* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-retail/minor/{retail-version}/pages/Use%20case%201%20-%20Customer%20profile%20sync/[Customer profile sync^] - The Customer profile sync use case enables retail developers to create a customer 360 view by unlocking customer data from disparate systems. 
+* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-service-cloud/minor/{service-cloud-version}/pages/Use%20case%201a%20-%20SAP%20to%20Salesforce%20Integration/[SAP to Salesforce Integration^] - Enable a single view of your customer by simplifying connectivity between SAP S/4 HANA and Salesforce.
```
Comment (sathya0) [inline service-cloud/1.0/modules/ROOT/pages/index.adoc]:
```suggestion
* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-service-cloud/minor/{service-cloud-version}/pages/Use%20case%201a%20-%20SAP%20to%20Salesforce%20Integration/[SAP to Salesforce Integration]
+
Enable a single view of your customer by simplifying connectivity between SAP S/4HANA and Salesforce.
```

**Review by nisha-nizamuddin — COMMENTED** [review]
LGTM

**Review by sathya0 — APPROVED** [review]

**Pull request merged**